src/offchainapi/asyncnet.py

```diff
@@ -128,7 +127,7 @@ def get_url(self, base_url, other_addr_str, other_is_server=False):
             server = self.vasp.get_vasp_address().as_str()
             client = other_addr_str
         url = f'v1/{server}/{client}/command/'
-        full_url = urljoin(base_url, url)
+        full_url = os.path.join(base_url, url)
```

Comment:
My onbly worry is that in UNIX systems path joins use the same "/" as URLs but this may not be true in general (like WIndows). Is that not a problem to us? Is there a more appropriate function in the urllib library for this purpose?